test/source.jl

```diff
@@ -439,3 +439,8 @@ ds = CSV.read(f)
 # #107
 df = CSV.read(IOBuffer("1,a,i\n2,b,ii\n3,c,iii"); datarow=1)
 @test size(df) == (3, 3)
+
+# #115
+df = CSV.read(joinpath(dir, "attenu.csv"), null="NA", rows_for_type_detect=200)
```

Comment:
BTW, `null` should really be renamed.

Reply:
WRT `null`, there's also a number of `nullable` occurrences. What's the current recommended term (adjective) for a fact that a container does/can store missing values?
For example, `MissingCategoricalArray` is a little bit misleading in that regard.

Reply:
I've not been able to come up with a good and short replacement for `nullable` so far... :-/

Reply:
[missable](http://www.yourdictionary.com/missable)?

Reply:
"Able to avoid detection."? :-)

I could find [one](https://rdrr.io/cran/rsolr/src/R/SolrExpression-class.R) precedent in relation with missing values, but that's the only one...